github/event_types.go

```diff
@@ -268,6 +268,22 @@ type LabelEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// MarketplacePurchaseEvent is triggered when a user purchases, cancels, or changes
+// their GitHub Marketplace plan.
+// Webhook event name "marketplace_purchase".
+//
+// Github API docs: https://developer.github.com/v3/activity/events/types/#marketplacepurchaseevent
+type MarketplacePurchaseEvent struct {
+	Action *string `json:"action,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	EffectiveDate               *Timestamp           `json:"effective_date"`
+	MarketplacePurchase         *MarketplacePurchase `json:"marketplace_purchase"`
+	PreviousMarketplacePurchase *MarketplacePurchase `json:"previous_marketplace_purchase"`
```

Comment:
The 3 fields above are missing `,omitempty` suffix. The rest of the fields have it.

You should add them for consistency (unless there's a good reason not to include them; I don't see one).